### ISSUE TITLE
Need to Reload Panel or Quit From Window

### ISSUE THREAD

**Issue by darkterminal** [usebruno/bruno#33]
When i launch bruno in my Linux Mint Desktop:

1. Renaming workspace (I need to quit from window and start bruno, then I can see workspace name is changed)
2. When I cancel to create a collection (I need to `ctrl+r` or click `view > reload` and my cursor work again)

Bruno version: v1.25.2 (On Desktop App)
File I downloaded is: `bruno_0.1.0_amd64_linux.deb` from Github Release Page

---
This project is amazing, if bruno have code request generator like Postman it will be nice :+1:  and `dark mode` theme.
Where I can read  `CONTRIBUTION.md` to know how it build and I can contribute to translate or documenting bruno.

**Comment by helloanoop** [comment]
Hey @darkterminal Thanks for your kind comments! You literally made my day today morning. 

I will get the 2 bugs fixed by today.
It's be awesome to support translations in different locale's (assuming thats what you meant) done for bruno.

I will have to setup a top level `TranslationProvider` component in react, and a `useTranslations` hook and read from the translations files while rendering the component.

I should be able to get this done in a day or two, but do let me know if you want to take a stab at this.

I haven't written up a contributing guide on how to get started, but here is a quick setup guide
We use React and Next.JS
```bash
# clone and cd into bruno
# use Node 14.x, Npm 8.x 

# Install deps (note that we use npm workspaces)
npm i

# run next app
npm run dev --workspace=packages/bruno-app

# open in browser
open http://localhost:3000
```

**Comment by helloanoop** [comment]
@darkterminal Both of these issue has been fixed in the latest release.
You can download the latest versions at https://www.usebruno.com/downloads

In case you love dark more, its included in the v0.2.0 release
Currently its based on vscode dark theme, but plan to support custom themes in the near future.

**Comment by darkterminal** [comment]
@helloanoop Thanks for your work! I did yet have time... so I will check it ASAP...

Well done!
